### PR TITLE
l10n | Enhancement

### PR DESCRIPTION
**Pull request by any-association**

### CODE REVIEW

File: localization/src/main/res/values-es-rES/strings.xml [hunk 1]
```diff
@@ -1151,7 +1151,7 @@
     <string name="multiplayer_space_stop_sharing">Dejar de compartir</string>
     <string name="multiplayer_space_stop_sharing_space">Dejar de compartir el espacio</string>
     <string name="multiplayer_space_stop_sharing_space_description">Los miembros ya no se sincronizarán con este espacio y el enlace compartido se desactivará.</string>
-    <string name="multiplayer_more_info">Más información</string>
+    <string name="multiplayer_more_info">More info</string>
```
Comment (migueldiab) [inline localization/src/main/res/values-es-rES/strings.xml:1154]: Why is this reverting to EN?
```suggestion
    <string name="multiplayer_more_info">Más info.</string>
```

File: localization/src/main/res/values-es-rES/strings.xml [hunk 2]
```diff
@@ -1179,14 +1179,24 @@
...
     <string name="multiplayer_notification_member_request_approved_with_edit_rights">Tu solicitud para unirte al espacio %1$s ha sido aprobada con derechos de acceso de edición. El espacio estará disponible en tu cuenta pronto.</string>
     <string name="multiplayer_notification_member_removed_from_space">Fuiste eliminado del espacio o el espacio ha sido eliminado por el propietario.</string>
+    <string name="multiplayer_notification_member_permissions_changed">Permissions changed</string>
     <string name="multiplayer_notification_member_permission_change_read">Se han cambiado tus derechos de acceso a solo lectura en el espacio %1$s.</string>
     <string name="multiplayer_notification_member_permission_change_edit">Se han cambiado tus derechos de acceso para editar en el espacio %1$s.</string>
     <string name="multiplayer_notification_member_join_request_declined">Tu solicitud para unirte al espacio %1$s ha sido rechazada.</string>
+    <string name="multiplayer_notification_request_declined">Request declined</string>
```
Comment (migueldiab) [inline localization/src/main/res/values-es-rES/strings.xml:1190]:
```suggestion
    <string name="multiplayer_notification_request_declined">Solicitud declinada</string>
```

File: localization/src/main/res/values-es-rES/strings.xml [hunk 3]
```diff
@@ -1179,14 +1179,24 @@
...
     <string name="multiplayer_notification_view_request">Ver</string>
+    <string name="multiplayer_notification_go_to_space">Ir al espacio</string>
     <string name="multiplayer_toast_permission_not_allowed">No está permitido cambiar permisos</string>
+    <string name="multiplayer_deeplink_to_your_object">Deeplink to your object</string>
```
Comment (migueldiab) [inline localization/src/main/res/values-es-rES/strings.xml:1194]:
```suggestion
    <string name="multiplayer_deeplink_to_your_object">Enlace-fuerte al objeto</string>
```

File: localization/src/main/res/values-es-rES/strings.xml [hunk 4]
```diff
@@ -1179,14 +1179,24 @@
...
+    <string name="multiplayer_deeplink_to_your_object_error">This deeplink is not working. Object might be deleted or there might be issues with space access.</string>
```
Comment (migueldiab) [inline localization/src/main/res/values-es-rES/strings.xml:1195]:
```suggestion
    <string name="multiplayer_deeplink_to_your_object_error">Este enlace-fuerte no funciona. El objeto puedo ser eliminado o puede haber inconvenientes con el acceso al espacio.</string>
```

**Review by uburoiubu — APPROVED** [review]

**Pull request merged**